manual_readme_content.md

```diff
@@ -0,0 +1,380 @@
+
+[![](img/slack_subscription_events.png)](img/slack_subscription_events.png)
+
+### Install App on Slack
+
+Next, click on **Install App** in the side bar. On that page, click **Install to Workspace**.
+
+[![](img/slack_install_app.png)](img/slack_install_app.png)
+
+On the next page, click **Allow**.
+
+[![](img/slack_allow_app.png)](img/slack_allow_app.png)
+
+After authorizing the app, the next window will show the app's authorization tokens. The **Bot User
+OAuth Access Token** will be required during asset configuration.
+
+[![](img/slack_auth_tokens.png)](img/slack_auth_tokens.png)
+
+## SOAR Configuration
+
+### SOAR Base URL
+
+The app uses the SOAR **Base URL** configuration to generate links to actions, so please make sure a
+valid url is specified in the **System Settings**.
+
+[![](img/slack_system_settings.png)](img/slack_system_settings.png)
+
+### Automation User Token
+
+The Slack Bot connector needs a SOAR authentication token to perform some tasks on the SOAR platform.
+To get this token, it is recommended that you create a new automation user. The steps for creating
+this user are as follows:
+
+-   On the SOAR platform, navigate to **Administration->User Management**
+-   Under **Users** , click **+ USER**
+-   In the **Add User** wizard, do the following:
+    -   Set the **User Type** to **Automation**
+    -   Give the user a **Username** like "Slack Automation"
+    -   For security reasons, accessing 127.0.0.1 is not allowed. Set **Allowed IPs** same as the
+        **instance IP or "any"**. (eg : If instance IP is 10.1.18.123, set allowed IP also
+        10.1.18.123)
+    -   Set the **Default Label** to the label seen in the Slack Bot asset's **Ingest Settings**
+    -   Under **Roles** , in addition to the default **Automation** role, add the **Observer** role
+    -   Click **CREATE**
+
+    [![](img/slack_automation_user.png)](img/slack_automation_user.png)
+
+-   Once the new user is created, click on the user in the user list
+-   On the user's page copy the **ph-auth-token** field from the **Authorization Configuration for
+    REST API** box
+
+    [![](img/slack_auth_token.png)](img/slack_auth_token.png)
+
+-   The token is required during asset configuration for the **Automation User Auth Token** field.
+
+### SOAR Slack Asset
+
+Fill out the required values in the **Asset Definition** tab.
+
+[![](img/slack_asset_info.png)](img/slack_asset_info.png)
+
+Fill out the **Bot User OAuth Access Token**, **Socket Token**, and **Automation User Auth Token**
+in the **Asset Settings** tab. See the prior sections for information on how to obtain each of these 
+tokens.
+
+[![](img/slack_asset_settings.png)](img/slack_asset_settings.png)
+
+Click **SAVE**. You will be asked to fill in the **Ingest Settings**. The "Label to apply
+to objects from this source" setting is ignored by this app, so it can be set to anything.
+
+Click **SAVE**.
+
+[![](img/slack_ingest_settings.png)](img/slack_ingest_settings.png)
```

Comment:
This image still has Phantom word in it. Is that how it still is in the platform

Reply:
nope, I just checked and it says Splunk SOAR now. this image is just old. let me update it

Reply:
Image updated in https://github.com/splunk-soar-connectors/slackbot/pull/7/commits/c7c77fbcf90cac315cbde3fc898228d888fa054c